enable `set -e` in `generate_protos.sh` in tlcc_enclave

**Describe the bug**
the script successfully completes without generating the needed source code.
the tlcc enclave makefile will later fail.

**To Reproduce**
Install a "wrong" protoc compile a run make in tlcc enclave!!!

**Expected behavior**
Script should fail immediately if protoc fails.

**Log-filesScreenshots**
Bad:
```
-- Configuring done
CMake Error at enclave/CMakeLists.txt:73 (add_library):
  Cannot find source file:

    protos/common/common.pb.c
```
Expected:
```
./generate_protos.sh enclave/protos
doing protos/fabric/common
common/common.proto:28:14: Missing numeric value for enum constant.
common/common.proto:29:14: Missing numeric value for enum constant.
Makefile:15: recipe for target 'enclave/protos' failed
make: *** [enclave/protos] Error 1

```

--- a/tlcc_enclave/generate_protos.sh
+++ b/tlcc_enclave/generate_protos.sh
@@ -4,7 +4,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-# set -eux
+set -e
 
 if [[ -z "${NANOPB_PATH}"  ]]; then
     echo "Error: NANOPB_PATH not set"

